Add template, form and view for Volunteer create.

maroshmka opened the pull request:
**Issue** - https://github.com/PythonBalkan/pyconbalkan/issues/46

_TODO:_
- add order model ❓  
- add photo upload ✅ 
- add form, view, etc ✅

katiayn commented:
@maroshmka about your questions: 
- I removed `weight` from `Volunteer`s model :) It was used to order the volunteers/speakers in a specific order but now it's gonna be ordered by `full_name`.
- You can add the `description` field to the form so the volunteers can fill this up :) 

Thx! 👍

katiayn commented on pyconbalkan/organizers/forms.py:
```diff
@@ -0,0 +1,22 @@
+
+    class Meta:
+        model = Volunteer
+        exclude = ('active', 'user', 'weight', 'type', 'slug', 'description', )
```
You can remove `weight`  (because I remove it) and `description` (so they can fill this up). 🔝😃

maroshmka replied:
done :)

maroshmka commented on pyconbalkan/organizers/views.py:
```diff
@@ -28,4 +29,21 @@ def organizers_listview(request):
+        if form.is_valid():
+            volunteer = form.save()
+            context['success'] = f'{volunteer.full_name}, you have been successfully signed up ' \
+                                 f'as a volunteer! Thank you!'
```
done :)

maroshmka commented:
Removed `weight`. Added photo upload. Some minor changes.

I didn't add order-field, cause when migrating, it want to set defaults for order attribute. And i guess there are some data already. But main problem is that in Volunteer model there are organizers + volunteers together, diffed by `type` attribute. But I guess we want to order only volunteers, or am I wrong? 

Possible solution (maybe in another PR?):
- add `TimeStampedModel` to `Person`, order by created_at + filter only volunteers
- split model Volunteer to - `Volunteer` and `Organizer`
- if we want to order also organizers - add order model + provide default + fix data manually.

what do you think @katiayn?

katiayn commented:
@maroshmka I already added ordering in in `Volunteer`, `Organizer` and `Speaker` models by `full_name`. Don't worry about the ordering for now.

Few more things:
- Plz, rebase to master: we had few changes in the meantime.
- I added `country` as a field in `Person`:
![image](https://user-images.githubusercontent.com/13651115/43977980-075bf24c-9ce6-11e8-901b-91e9673ce1c1.png)
can you fix the field, plz?
- check some comments I'll leave here.

Thanks! 😃

katiayn commented on pyconbalkan/urls.py:
```diff
@@ -44,6 +44,7 @@
     path('robots.txt', TemplateView.as_view(template_name='robots.txt', content_type='text/plain')),
     path('organizers/<slug:slug>/', organizer_view, name='organizer_detail'),
     path('organizers', organizers_listview, name='organizers'),
+    path('volunteer/create/', volunteer_createview, name='volunteer_create'),
```
@maroshmka let's use `volunteers` (in plural) to keep the consistence with organizers url. :)

maroshmka replied:
done

maroshmka commented:
rebased and changed urls. 

Dont know how to round that select (simple border-radius wont work), it would be faster if someone who knows frontend better than I would look at it.

katiayn commented:
@maroshmka no problem :) I'll ask some frontend to check it out. 

Anyway, thanks! 👍

katiayn commented:
@maroshmka I changed the link to the form because now they want to have a google form with questions and after they select people, they will fill the volunteers form in the website - the one you created. 😃 
They also asked me to remove the button from the main page and add to the organizers page only - for now.

Thanks for your help!

maroshmka commented:
alright :) np
